keg_login/middleware.py

```diff
@@ -0,0 +1,27 @@
+import flask
+
+
+class LockedOutMiddleware(object):
+    lock_endpoint = 'keg_login.lock'
+
+    session_key_name = 'keg-login.lockout'
+
+    default_excluded_routes = [
+        'static',
+        'keg_login.logout',
+        lock_endpoint,
+    ]
+
+    def __init__(self, exclude=[]):
+        self.exclude_routes = exclude + self.default_excluded_routes
```

Comment:
It's more than a NIT IMO.  You should never have a mutable default argument like this.  In this case, the way it's used doesn't matter.  But I don't even want the possibility of that coming back to bite us later.